content/articles/polymorphism-in-java/index.md

```diff
@@ -45,14 +51,16 @@ Polymorphism can be divided into two:
 
 #### 1. Static Polymorphism
 
-> **Static Polymorphism** - When you are writing a code, probably creating a method, and you by chance create two or more methods with the same name, you will get an alert that you have a duplicate method in your code. This error is called `Compile-time error`. The ability to receive alerts on errors in your code is made possible by the **compiler**. When the compiler resolves conflict on matters regarding polymorphism, it is regarded as `Compile-time Polymorphism`
-Examples include Overloading also known as Method overloading or Function overloading.
+> **Static Polymorphism** is the linking of a function or a method with the object when compiling the code. This is before running the code.
+To understand what compile-time error and compile-time polymorphism is, let's take a case. When you are writing a code, probably creating a method, and you by chance create two or more methods with the same name. You will get an alert that you have a duplicate method in your code. This error is called `Compile-time error`. The ability to receive alerts on errors in your code is made possible by the **compiler**. When the compiler resolves conflict on matters regarding polymorphism, it is regarded as `Compile-time Polymorphism`.
+
+An example includes Method overloading or Function overloading.
 
 For clarity:
 
-`Method overloading` is achieved by having multiple methods to have the same name but different parameters. The parameters can differ by having different arguments, having different types of data types, or having different sizes the arguments. This will avoid the compile-time error.
+`Method overloading` is achieved by having multiple methods to have the same name but different parameters or having different sizes of arguments. The parameters can differ by having different arguments, having different types of data types, or having different sizes the arguments. This will avoid the compile-time error.
 
-Let us use a code snippet to try to understand more about static polymorphism.
+Let us use a code snippet understand more about static polymorphism.
```

Comment:
Let us use a code snippet to understand more about static polymorphism.

Reply:
Done